src/components/GroupSystems/GroupSystems.cy.js

```diff
@@ -229,14 +229,14 @@ describe('filtering', () => {
         applyNameFilter();
         cy.wait('@getHosts')
         .its('request.url')
-        .should('include', 'hostname_or_id=lorem');
+        .should('include', 'name=lorem');
     });
 
     it('can remove the chip or reset filters', () => {
         applyNameFilter();
         cy.wait('@getHosts')
         .its('request.url')
-        .should('contain', 'hostname_or_id=lorem');
```

Comment:
In my case, it's failing here. If I change it back to 'hostname_or_id=lorem' it's passing.